frontend-ssr/.gitignore

```diff
@@ -22,9 +22,11 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock*
```

Comment:
We must not ignore this one. The `yarn.lock` file makes sure we all install the exact same dependencies, meaning exact same version. Otherwise , different `yarn` commands may have different end results in the `node_modules`.